### ISSUE TITLE
Problem useing "horizontal-stack" or "vertical-stack"  inside the "vertical-stack-in-card"  

### ISSUE THREAD

**Issue by VirtualL** [custom-cards/vertical-stack-in-card#13]
Hi , I got a problem with using horizontal-stack **or** vertical-stack  inside the custom card , 
the warning i got:

>  **2018-08-06 12:38:57 ERROR (MainThread) [frontend.js.latest.201808040] https://MYSERVER.COM/local/custom_components/vertical-stack-in-card.js?v=0.0.4:58:38 Uncaught ReferenceError: root is not defined**

my config is : 
```
views:
#### FIRST view ####
  - title: Home
    id: Home
    cards:
      - type: custom:vertical-stack-in-card
        cards:
          - type: entities
            entities:
              - group.allmylights
          ##
          - type: horizontal-stack
            cards:
              - type: entities
                show_header_toggle: false
                entities:
                  - light.white_bulb
                  - light.living_room_color_bulb
                  - type: divider
                  - light.sonoff_hallway_light
                  - light.gateway_light_34ce0088d2f7
              - type: entities
                show_header_toggle: false
                entities:
                  - light.sonoff_light
                  - light.philips_bulb
                  - type: divider
                  - light.desk_lamp
                  - switch.bedroom_fan_light
                  
          ### all my lights in Vertical stak ####   
          - type: vertical-stack
            cards:
              - type: horizontal-stack
                cards:
                  - type: picture-glance
                    image: /local/MyEntities/livingroom.jpeg
                    entities:
                      - light.white_bulb
                      - light.living_room_color_bulb
                  - type: picture-glance
                    image: /local/MyEntities/kitchen.jpeg
                    entities:
                      - light.sonoff_light
                      - light.philips_bulb
                ######      
              - type: horizontal-stack
                cards:
                
                  - type: picture-glance
                    image: /local/MyEntities/hallway.jpeg
                    entities:
                      - light.sonoff_hallway_light
                      - light.gateway_light_34ce0088d2f7
                  - type: picture-glance
                    image: /local/MyEntities/bedroom.jpeg
                    entities:
                      - light.desk_lamp
                      - switch.bedroom_fan_light
```
    
its also appears when I use "vertical-stack" only, or "horizontal-stack" only

its look like the place the problem appears is here:
```
    connectedCallback() {
        this._refCards.forEach((element) => {
            if (element.shadowRoot) {
                if (!element.shadowRoot.querySelector('ha-card')) {
                    var searchEles = root.childNodes[index].shadowRoot.getElementById("root");
                    if (!searchEles) {
                        searchEles = root.childNodes[index].shadowRoot.getElementById("card");
                    }
                    searchEles = searchEles.childNodes;
                    for(var i = 0; i < searchEles.length; i++) {
                        searchEles[i].style.margin = "0px";
                        searchEles[i].shadowRoot.querySelector('ha-card').style.boxShadow = 'none';
                        searchEles[i].shadowRoot.querySelector('ha-card').style.paddingBottom = '0px';
                    }
                } else {
                    element.shadowRoot.querySelector('ha-card').style.boxShadow = 'none';
                    element.shadowRoot.querySelector('ha-card').style.paddingBottom = '0px';
                    element.shadowRoot.querySelector('ha-card').style.paddingTop = '0px';
                }
            }
        });
    }

```

_I am Using HA 0.75.1 , vertical-stack-in-card V 0.0.4 (out few minute ago) , and the problem in Safari latest , and Chorme latest_

**P.S** before V 0.0.4 of vertical-stack-in-card when i was try to do it , all the frontend was freeze , with  logs filled up every click i do.

**Comment by ofekashery** [comment]
Hi Eliran, Thanks for the report!
Fixed in new version (0.0.5).

**Comment by VirtualL** [comment]
thank you
